contrib/examples/apiserver/broker.yaml

```diff
@@ -8,6 +8,7 @@ spec:
   # service-catalog will use the contents of the secret. The secret should have "username"
   # and "password" keys
   authInfo:
-    basicAuthSecret:
+    secretRef:
```

Comment:
this is an API break, so we should let people know that this is going to change.